ci-operator/config/openshift/cluster-api-provider-kubevirt/openshift-cluster-api-provider-kubevirt-main.yaml

```diff
@@ -168,10 +168,10 @@ tests:
       commands: |-
         set -x
 
-        export CRI_PATH: /var/run/crio/crio.sock
-        export IMAGE_REPO: k8s.gcr.io
-        export TENANT_CLUSTER_KUBERNETES_VERSION: v1.21.0
-        export NODE_VM_IMAGE_TEMPLATE: quay.io/kubevirtci/fedora-kubeadm:35
+        export CRI_PATH=/var/run/crio/crio.sock
+        export IMAGE_REPO=k8s.gcr.io
+        export TENANT_CLUSTER_KUBERNETES_VERSION=v1.21.0
+        export NODE_VM_IMAGE_TEMPLATE=:wquay.io/kubevirtci/fedora-kubeadm:35
```

Comment:
Remove the `:w` after the `=` in this line

Reply:
Thanks @nirarg - fixed.